models/public/yolo-v1-tiny-tf/yolo-v1-tiny-tf.md

```diff
@@ -0,0 +1,96 @@
+# yolo-v1-tiny-tf
+
+## Use Case and High-Level Description
+
+YOLO v1 Tiny is a real-time object detection model from TensorFlow JS\* framework. This model was pretrained on VOC\* dataset with 20 classes.
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 6.988         |
+| MParams           | 15.858        |
+| Source framework  | TensorFlow\*  |
+
+## Input
+
+### Original model
+
+Image, name - ``, shape - `1,416,416,3`, format is `B,H,W,C` where:
+
+- `B` - batch size
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`.
+Scale value - 255.
+
+### Converted model
+
+Image, name - ``, shape - `1,3,416,416`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Output
+
+### Original model
+
+The array of detection summary info, name - `conv2d_9/BiasAdd`,  shape - `1,13,13,125`, format is `N,Cx,Cy,B*5` where
```

Comment:
> B*5

Doesn't it make more sense to describe it as `B*25`, where B is the _number_ of boxes?